### PR TITLE
LPS-134450 Add test to check if the checkbox and the kebab is being disabled

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageItem.es.js [hunk 1]
```diff
@@ -52,8 +52,10 @@ function Item({totalCount, ...instance}) {
 	const assignedToUser = !!assignees.find(({id}) => id === Number(userId));
 	const assigneeNames = assignees.map((user) => user.name).join(', ');
 	const {reviewer} = assignees.find(({id}) => id === -1) || {};
+	const hasTasksWithoutAssignee = !!assignees.find(({id}) => id === -1);
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/InstanceListPageItem.es.js:55]: Hey @seltong, the variable `hasTasksWithoutAssignee` is not required. You are able to achieve the same result using the `reviewer`.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPageItem.es.js [hunk 2]
```diff
@@ -468,6 +468,33 @@ describe('The InstanceListPageItem instance checkbox component should', () => {
 		taskNames: ['Review'],
 	};
 
+	test('Disable the checkbox when the id is different of the userId', () => {
+		const {container} = render(
+			<Table.Item {...{...instance, id: 12345}} />,
```
Comment (jotascript) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/test/components/instance-list-page/InstanceListPageItem.es.js:473]: Hi @seltong , I dont see a problem with this... 
But In this line and 487 line, you can do it like this: `<Table.Item {...instance} id={12345} />`
Remember that "explicit is better than implicit".

**Comment by ramonmsp** [comment]
Good Job :D

**Review by ramonmsp — APPROVED** [review]

**Pull request closed without merging**